p3-frontend/package-lock.json

```diff
@@ -1,16371 +1,8 @@
 {
   "name": "p3-frontend",
   "version": "0.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
by running npm install using npm 6 (last time i checked, still the default even with the latest version of node.js), you changed the format of this lock file. since it was version 2, the devs must have been using npm 7. this is, strictly speaking, something that should be taken into account in the pipeline even though it's unlikely to cause practical issues.
work out how to get the pipeline to run npm ci with npm 7, and let me know if you run into trouble with that.
also, revert the huge diff to this file, which you should have already noticed and asked about or investigated.

Reply:
reverted file to version found in main.